packages/schema/lib/schemas/OutputFieldSchema.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+const FieldSchema = require('./FieldSchema');
+
+module.exports = makeSchema(
+  {
+    description: 'Field schema specialized for output fields.',
```

Comment:
```suggestion
    description: 'Field schema specialized for output fields. ${FieldSchema.schema.description}',
```

Based on this suggestion https://github.com/zapier/zapier-platform/pull/957/files#r1940592149